src/components/Tool/Appeal/InitialPage/AddAppealForm.tsx

```diff
@@ -29,8 +29,8 @@ import {
 } from 'pages/accountLists/[accountListId]/tools/GetAppeals.generated';
 import { MultiselectFilter } from 'src/graphql/types.generated';
 import i18n from 'src/lib/i18n';
-import theme from '../../../theme';
-import AnimatedCard from '../../AnimatedCard';
+import theme from '../../../../theme';
+import AnimatedCard from '../../../AnimatedCard';
 import { useCreateAppealMutation } from './CreateAppeal.generated';
 import { useGetContactTagsQuery } from './GetContactTags.generated';
 
```

Comment:
on this component, could you change "Select Some Options" to "Select some options"?
<img width="540" alt="Screenshot 2024-08-05 at 2 35 05 PM" src="https://github.com/user-attachments/assets/86b9f5da-a105-43d6-8ca4-88394e0e86c5">